core/server/master/src/main/java/alluxio/master/metrics/DefaultMetricsMaster.java

```diff
@@ -141,9 +141,11 @@ protected void registerThroughputGauge(String counterName, String throughputName
         new Gauge<Object>() {
           @Override
           public Object getValue() {
-            long uptime = (System.currentTimeMillis() - mMetricsStore.getLastClearTime())
-                / Constants.SECOND_MS;
-            return uptime == 0 ? 0 : MetricsSystem.counter(counterName).getCount() / uptime;
+            long uptime = (mClock.millis() - mMetricsStore.getLastClearTime())
+                / Constants.MINUTE_MS;
+            long value = MetricsSystem.counter(counterName).getCount();
+            // The value is bytes per minute
+            return uptime == 0 ? value : value / uptime;
```

Comment:
hmm if uptime is 0 should we be returning `value`? or `0`?

In the off chance of some kind of error, what if uptime is negative?

Reply:
Oh, that should not happen. mClock.mills() should always be equal or bigger than the mMetricsStore.getLastClearTime()). 
I feel returning value in the first minute actually makes more sense. I am fine with both 0 or value just feeling that using value can even provide users information in the first minute so it's better.

Reply:
wait, actually uptime can be negative..... 
Let me modify the code

Reply:
Changed the code to make sure uptime is positive. Thanks for finding out that!!